packages/web-react/src/components/Link/__tests__/useLinkStyleProps.test.ts

```diff
@@ -10,4 +10,18 @@ describe('useLinkStyleProps', () => {
 
     expect(result.current.classProps).toBe(expectedClassName);
   });
+
+  it('should return link-underline class', () => {
+    const props = { color: 'primary', underline: 'always' } as SpiritLinkProps;
+    const { result } = renderHook(() => useLinkStyleProps(props));
+
+    expect(result.current.classProps).toContain('link-underlined');
+  });
+
+  it('should return link-not-underline class', () => {
+    const props = { color: 'primary', underline: 'never' } as SpiritLinkProps;
+    const { result } = renderHook(() => useLinkStyleProps(props));
+
+    expect(result.current.classProps).toContain('link-not-underlined');
```

Comment:
same here. the test is called `...underline...` but it expects `...underlined...`